src/CheckoutSdk/CardIssuing/CardHolders/CardHoldersClient.cs

```diff
@@ -0,0 +1,55 @@
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Checkout.CardIssuing.CardHolders
+{
+    public class CardHoldersClient : AbstractClient, ICardHoldersClient
```

Comment:
Are we sure that implementing separated clients is a good approach for this? We will have a `CardholdersClient`, `CardsClient`, `CardControlsClient` and then `CardsTestingClient` when it's all related to CardIssuing. In Node we only have `IssuingClient`. WDYT @armando-ibarra-cko ?

Reply:
I mean, we could still separate domain classes in packages, but I think it would be better to have one client for everything

Reply:
Yes, for the client, issuingClient for all is the best implementation.